scripts/fix-engine-names.py

```diff
@@ -154,13 +172,15 @@ def fixVariousVersions(t):
             "LCZeroCPU_3pct",
             "Rodent",
             "Sjeng",
+            "SlowChess_Blitz",
             "Stoofvlees",
             "Wasp",
             "Zappa",
         ]:
             name = n
             version = v + " " + version
             version = version.replace(" ", "_")
+    name = name.replace("SlowChess_Blitz", "SlowChess Blitz")
```

Comment:
Just to reiterate. This in general is dubious programming practice with these temporary substitutions and then back. In general, if one ever needs to do this, it is a sign that there is an underlying problem that should be fixed, instead.

Reply:
The problem is that these things tend to become hard to maintain.